pkg/cmd/open/open.go

```diff
@@ -115,7 +124,10 @@ func runOpenCmd(opts *OpenOptions) error {
 		}
 
 		for _, shortcut := range shortcuts {
-			url := nameURLmap[shortcut]
+			url := nameURLmap[shortcut].Default
+			if applicationID != "" {
+				url = nameURLmap[shortcut].WithAppId
+			}
```

Comment:
We have this logic at two different place now, I guess we could build the list of shortcuts before (using the AppID) and re-use it here and below?

Reply:
Also, I don't think the test at the line #105 is necessary anymore (adding the `/` before the appID)